app/Models/Action.php

```diff
@@ -36,7 +36,7 @@ class Action extends Model
      *
      * @var array
      */
-    public static $indexes = ['id'];
+    public static $indexes = ['id', 'callpower_campaign_id'];
```

Comment:
I don't believe this column has an index in the database. Can we add one as part of this PR?